cetmix_tower_aws/readme/USAGE.md

```
@@ -0,0 +1,112 @@
+        result["message"] = "Found " + str(len(instance_details)) + " EC2 instances:\n" + "\n".join(instance_details)
+    else:
+        result["message"] = "No EC2 instances found"
+    ```
+
+* **Save the Command**
+   - Click the `Save` button to create the command
+
+**Running the AWS EC2 Command**
+
+* **Navigate to Server**
+   - Go to `Cetmix Tower > Servers > Servers`
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)